src/cell.js

```diff
@@ -181,6 +181,7 @@ export class Cell {
         }
         this.zoomed = !this.zoomed
         this.gate.sendState()
-        this.t7.run(() => this.focus(), ABIT)
+        if (focus)
```

Comment:
I'm having trouble imaging a user scenario where they zoom in/out a pane and don't want it focus. 
What am I missing

Reply:
Well most of the times we call the function with focus=true but if we want to close a pane and go home and thus need to unzoom we can't have the pane focus since Cell.focus() toggles divide button and changes the URL.
A better way might be to move it to a different Cell.unzoom() function.